sdcm/cluster.py

```diff
@@ -439,7 +442,7 @@ def init(self) -> None:
         self.start_task_threads()
         self._init_port_mapping()
 
-        self.set_keep_alive()
+        self.set_keep_alive(until=datetime.utcnow() + timedelta(hours=KEEP_ALIVE_ON_FAILURE, minutes=TEST_DURATION))
```

Comment:
why add both 72 and the duration of the test ?

Reply:
Because it's 72h _after_ test complete.